assets/css/main.css

```diff
@@ -35,19 +36,23 @@
     padding: 0 16px;
 }
 
-.pagination .next {
-    margin-left: auto;
```

Comment:
Why remove this, while you can add ` margin-inline-start: auto;` to this too ?

Reply:
I think [this part](https://github.com/adityatelange/hugo-PaperMod/pull/47/files#diff-b15932692c619f9a334ecb156ac167557c3af214a8a3df8990c09232ac8a5baaR27) does its job in a better way.

Reply:
The next button is aligned to left hand side when there is no previous button. It could be as it was before.

Reply:
Oh! You're right! :)) I'll fix it now.

Reply:
Done!